webpage.py

```diff
@@ -82,8 +82,8 @@ def __init__(self, name: str, kills: int, losses: int, streak: int):
 @router.get("/400")
 @aiohttp_jinja2.template("400.jinja2")
 async def challenge(req: web.Request):
-    from cache.year_run_stats import get_run_stats # TODO: Fix circular imports with router
-    run_stats = get_run_stats()
+    from cache.run_stats import get_all_run_stats # TODO: Fix circular imports with router
+    run_stats = get_all_run_stats()
```

Comment:
oh, we COULD update this to get only 2022 now if we cared at all

Reply:
Yeah, could do, but I'm not sure we even use this page anymore... If it's easy, no harm